docs/guides/connecting_to_your_data/datasource_configuration/sql_components/_table_splitting_methods.mdx

```diff
@@ -0,0 +1,23 @@
+
+:::note NOTES ON SPLITTING METHODS
+
+- The names of `splitting_method` values can be specified with or without a preceding underscore.
+
+:::
+
+#### Available methods for `splitting_method` values and their configuration parameters:
+
+| `splitting_method`                | `splitting_kwargs`                                                       | Returned Batch Data                                                                                                                                                                                           |
```

Comment:
```suggestion
| `splitter_method`                | `splitter_kwargs`                                                       | Returned Batch Data                                                                                                                                                                                           |
```

Reply:
I corrected a few more instances in that section where I had used `*ing_*` instead of `*er_*`.